src/data_preprocessing/monographs/monograph_scrapper.py

```diff
@@ -0,0 +1,17 @@
+import os
+from pathlib import Path
+import urllib.request
+
+
+def scrap_monographs(urls, data_path):
```

Comment:
Compare: https://en.wiktionary.org/wiki/scrap#Verb
with: https://en.wiktionary.org/wiki/scrape#Verb
Notice the vastly different meaning and pronunciation.

Please fix the filename as well.